egs/aishell/asr1/run.sh

```diff
@@ -178,6 +182,26 @@ if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
         --dict ${dict}
 fi
 
+ngramexpname=train_ngram
```

Comment:
Can you include this ngram LM training in stage 3?
I want to keep the role of the stage.

Reply:
done